src/bvals/comms/bnd_info.cpp

```diff
@@ -177,7 +177,7 @@ CalcIndices(const NeighborBlock &nb, MeshBlock *pmb,
                                 (neighbor_bounds[dir].e - neighbor_bounds[dir].s + 1);
         s[dir] += nb.origin_loc.l(dir) % 2 == 1 ? extra_zones - interior_offset : 0;
         e[dir] -= nb.origin_loc.l(dir) % 2 == 0 ? extra_zones - interior_offset : 0;
-        if (ir_type == IndexRangeType::InteriorSend) {
+        if (ir_type == IndexRangeType::InteriorSend && !prores) {
```

Comment:
Changes to `CalcIndices` are required since we are actually passing the correct neighbor information in `GetInterior*`.